package.json

```diff
@@ -3,13 +3,13 @@
   "version": "1.1.0",
   "description": "Retrieve and display list of recommended benefits available to Californians",
   "main": "dist/index.js",
+  "type": "module",
   "scripts": {
     "build:js": "esbuild src/index.js --bundle --outfile=dist/index.js --loader:.html=text --loader:.css=text --format=esm",
     "build:more-args": " --minify ",
-    "build": "sass src/css/index.scss src/css/index.css && npm run build:js",
```

Comment:
I love sass. :(

Reply:
Would having a watch that auto builds and refreshes on change make sass less annoying for you?